src/command-abstractions/interaction-base.ts

```diff
@@ -1,8 +1,8 @@
 import { strict as assert } from "assert";
 import { ConditionalOptional } from "../utils/typing.js";
 
-export const ApplicationCommandTypeUser = 2;
-export const ApplicationCommandTypeMessage = 3;
+export const APPLICATION_COMMAND_TYPE_USER = 2;
+export const APPLICATION_COMMAND_TYPE_MESSAGE = 3;
```

Comment:
This is a tricky one. I went with the `ApplicationCommandTypeMessage` because this is how it appears in the discord.js docs https://discord-api-types.dev/api/discord-api-types-v10/enum/ApplicationCommandType (for some reason I wasn't able to import it, if memory serves). It might be best to eslint-ignore these two.

Reply:
I just reverted that change and added `// @eslint-ignore-next-line`